ocp_resources/network_attachment_definition.py

```diff
@@ -197,3 +197,39 @@ def to_dict(self):
     @property
     def resource_name(self):
         return f"ovs-cni.network.kubevirt.io/{self.bridge_name}"
+
+
+class OVNOverlayNetworkAttachmentDefinition(NetworkAttachmentDefinition):
+    def __init__(
+        self,
+        network_name=None,
+        *args,
```

Comment:
we limit it only to `kwargs`, remove `args`

Reply:
Ack